### PR TITLE
chore: improve import order and disable autoImport on dist folder

### PR DESCRIPTION
**Pull request by VincentMolinie**
### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

**Comment by VincentMolinie** [comment]
The autofix on import order did not work really well before, with this fix there is no issue anymore

File: .eslintrc.json [hunk 1]
```diff
@@ -10,49 +10,87 @@
     "plugin:@typescript-eslint/recommended",
     "plugin:prettier/recommended",
     "plugin:jest/recommended"
+    // "plugin:import/recommended"
```
Comment (Scra3) [inline .eslintrc.json:13]: Why it is commented ? :)

File: .vscode/settings.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
   "debug.javascript.terminalOptions": {
     "skipFiles": ["<node_internals>/**", "**/node_modules"]
   },
+  "typescript.preferences.autoImportFileExcludePatterns": ["**/dist/*"],
```
Comment (Scra3) [inline .vscode/settings.json:25]: 👍

File: .eslintrc.json (hunk 1 above)
Comment (VincentMolinie) [reply .eslintrc.json:13]: Should be removed sorry

**Review by Scra3 — APPROVED** [review]

**Pull request merged**